resources/aws/bucket.go

```diff
@@ -8,13 +8,18 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	microerror "github.com/giantswarm/microkit/error"
 	"github.com/juju/errgo"
+
+	"github.com/giantswarm/aws-operator/resources"
 )
 
 type Bucket struct {
 	Name string
 	AWSEntity
 }
 
+// Implement ReusableResource.
+var _ = resources.ReusableResource(&Bucket{})
```

Comment:
For now maybe we are not using those interfaces anywhere, but one of the biggest concerns we had some time ago was abstracting resources (i.e. for potential re-usage in operatorkit) and lack of uniqueness in resource methods.